javascript2-week2/quizApp2.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Quiz App</title>
+    <link rel="stylesheet" type="text/css" href="quizApp2.css">
+</head>
+
+<body>
+    <div class="quiz-form">
+        <h2>Add a Quiz Question</h2>
+        <form id="quizForm">
+            <div class="form-group">
+                <label for="question">Question (max 140 characters):</label>
+                <input type="text" id="question" maxlength="140" required>
+            </div>
+            <div class="options">
+                <label>Options:</label>
```

Comment:
Each input should have it's own label. If you dont want them to be visible you can add `aria-label="Option 1"` attribute to each input, although it is generally better to leave them visinble. (You can read more about labelling here)[https://css-tricks.com/html-inputs-and-labels-a-love-story/] This is for accessibility reasons - when a person that has trouble with sight is usung your webpage they oftem may use a screen reader. For screen reader to read the for properly each input has to be labelled.